HelpSource/Tutorials/A-Practical-Guide/PG_Cookbook05_Using_Samples.schelp

```diff
@@ -20,10 +61,13 @@ code::
 b = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
 
 // one loop segment
-SynthDef(\oneLoop, { |out, bufnum, start, time, amp|
-	var	sig = PlayBuf.ar(1, bufnum, startPos: start, loop: 0),
-		env = EnvGen.kr(Env.linen(0.01, time, 0.05, level: amp), doneAction: Done.freeSelf);
-	Out.ar(out, (sig * env) ! 2)
+SynthDef(\oneLoop, { |out, bufnum, start, time, amp, rate = 1|
+	var	sig = PlayBuf.ar(1, bufnum,
```

Comment:
It looks like there's a tab between `var` and `sig = ...`. This makes for some strange indentation in the HelpBrowser. Would you mind replacing them with a space here and elsewhere?

Reply:
OK, fixed.